src/Controller/Admin/UserController.php

```diff
@@ -69,12 +70,23 @@ public function edit(Request $request, User $user, EntityManagerInterface $entit
     }
 
     /**
-     * @Route("/{id}", name="delete", methods={"POST"})
+     * @Route("/{id}", name="delete")
      */
     public function delete(Request $request, User $user, EntityManagerInterface $entityManager): Response
     {
             $entityManager->remove($user);
             $entityManager->flush();
         return $this->redirectToRoute('admin_user_index');
     }
+
+    /**
+     * @Route("/list/unactivate", name="unactivate")
```

Comment:
Admin index needs to be changed ('Hello admincontroller") in order to add a button and an explanation text of this method.